config/200-serviceaccount.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: 3scale-kourier
+  name: net-kourier
```

Comment:
Curious any reason why you don't opt into the aggregated cluster role?

Reply:
I think no reason. We can use aggregated cluster role.